main.py

```diff
@@ -33,12 +34,13 @@ def parse_arguments():
                         help='File patterns to include (default: *.jpg *.jpeg *.png, *.JPG, *.JPEG, *.PNG')
     parser.add_argument('--exclude', nargs='+', default=["*_border*"],
                         help='File patterns to exclude (default: *_border*)')
-    parser.add_argument('-f', '--font', default='Roboto-Regular.ttf', 
+    parser.add_argument('-f', '--font', default='Roboto-Regular.ttf',
                         help='Font file in fonts directory')
-    parser.add_argument('-fb', '--fontbold', default='Roboto-Medium.ttf', 
+    parser.add_argument('-fb', '--fontbold', default='Roboto-Medium.ttf',
                         help='Bold font file in fonts directory')
     return parser.parse_args()
 
+
 def process_image(path: str, add_exif: bool, add_palette: bool, border_type: BorderType, fontname: str, boldfontname: str) -> str:
```

Comment:
Should this be `fontpath` & `boldfontpath` now?

Reply:
So, it's still the font name at this  point right? The process_image function then determines the paths within the exif if statement.